### PR TITLE
Feature/us18380 auth library for profile

### PR DESCRIPTION
**Pull request by vchtelmakh**
---

### CODE REVIEW

File: src/app/giving/recurring/recurring.component.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 <div *ngIf="!givingService.loading && (!recurringGifts || recurringGifts.length < 1)">
   <p class="push-half-top callout callout-info">
-    <crds-content-block id="pushpayNoRecurringGifts"></crds-content-block>
+    <!-- <crds-content-block id="pushpayNoRecurringGifts"></crds-content-block> -->
```
Comment (vchtelmakh) [inline src/app/giving/recurring/recurring.component.html]: @briancavanaugh is working on this, we will need to bring it back in once it's updated before we merge this.

File: src/app/app.contentblock.config.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import { ContentBlockConfig } from 'crds-ng2-content-block';
+//import { ContentBlockConfig } from 'crds-ng2-content-block';
 
 export function contentBlockFactory() {
 
-  const contentBlockConfig: ContentBlockConfig = new ContentBlockConfig(
-    environment.cmsEndpoint + '/',
-    ['common', 'giving', 'main']
-  );
+  // const contentBlockConfig: ContentBlockConfig = new ContentBlockConfig(
```
Comment (jpc11623) [inline src/app/app.contentblock.config.ts]: Commented code

File: inject-env-vars.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+replaceAngularEnvValue() {
+  echo "Setting $1 to $2 in $3"
+  sed -E -i "s/^([[:blank:]]*OKTA_CLIENT_ID[[:blank:]]*:[[:blank:]]*).*/\1'$2',/" ./src/environments/environment.$3.ts
+}
+
+echo "Branch is: ${BRANCH}"
+
+if [ "${BRANCH}" == "master" ]; then
+  echo "Setting Secrets for PRODUCTION"
+  replaceAngularEnvValue OKTA_CLIENT_ID "$OKTA_CLIENT_ID_SANDBOX" prod
```
Comment (vchtelmakh) [inline inject-env-vars.sh]: Needs to be `$OKTA_CLIENT_ID` - this is my bad, had it wrong in the thing I linked.

Comment (candrews55) [reply inject-env-vars.sh]: Done

File: netlify.toml [hunk 4]
```diff
@@ -17,12 +17,12 @@
   CRDS_ENV = "int"
 
 [context.branch-deploy]
-  command = "npm run test:ci && npm run build:simple && npm run env:replace && node ./cypress/kickOffCypress.js"
+  command = "npm run test:ci && npm run build:simple && chmod +x ./inject-env-vars.sh && node ./cypress/kickOffCypress.js"
```
Comment (vchtelmakh) [inline netlify.toml]: All these need `chmod +x ./inject-env-vars.sh && npm run inject-env-vars` - you are allowing it, but not running it.

Comment (candrews55) [reply netlify.toml]: Done

**Pull request merged**